cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ProcessConfiguration.java

```diff
@@ -17,11 +18,20 @@
     protected int maxHeapMemorySize = -1;
 
     protected Collection<String> jvmOptions;
+    protected Collection<String> processParameters = new ArrayList<>();
+
 
     public ProcessConfiguration(ServiceEnvironmentType environment, int maxHeapMemorySize, Collection<String> jvmOptions) {
+        this(environment, maxHeapMemorySize, jvmOptions, new ArrayList<>());
+    }
+
+    public ProcessConfiguration(ServiceEnvironmentType environment, int maxHeapMemorySize, Collection<String> jvmOptions, Collection<String> processParameters) {
         this.environment = environment;
         this.maxHeapMemorySize = maxHeapMemorySize;
         this.jvmOptions = jvmOptions;
+
+        if(processParameters != null)
```

Comment:
`if (` and always use braces on if statements.

Reply:
fixed